src/frontend/next/src/styles/globals.css

```diff
@@ -1,16 +1,145 @@
-html,
-body {
-  padding: 0;
+/* As pages/_app.tsx import this CSS it will resets the CSS
+* and makes our <html> font 10px.  Any other "global" things
+* that need to happen for a page should get added here, and
+* then they will trickle down to the other pages.
+ */
+
+html {
+  padding: none;
+  margin: none;
+  font-size: 10px;
+}
+
+/* -------------------------------------------------- */
+
+/* telescope-post-content.css */
```

Comment:
Instead of doing this inside `global.css`, can you move this to its own file: `src/frontend/next/src/styles/telescope-post-content.css` and do an `@import` here to include it.

We need to do the same with the highlight-js CSS stylesheet for our code formatting.

Reply:
Sure.

Reply:
@humphd I could not find the file `highlight-js` could you please provide the path to me?

Reply:
We need to install the highlightjs package to next/, then pull the CSS file from node_modules:

```
    "highlight.js": "10.4.1",
```

Then we can use `node_modules/highlight.js/styles/github.css`

Reply:
Okay. Let's do it in a follow-up.